src/widgets/list.rs

```diff
@@ -211,26 +255,36 @@ impl<'a> StatefulWidget for List<'a> {
         {
             let (x, y) = match self.start_corner {
                 Corner::BottomLeft => {
-                    current_height += item.height() as u16;
+                    current_height += item.height() as u16 + self.padding as u16;
                     (list_area.left(), list_area.bottom() - current_height)
```

Comment:
:lady_beetle:  :lady_beetle:  :lady_beetle: 
`thread 'main' panicked at 'attempt to substruct with overflow', src/widgets/list.rs:259:40`

Reply:
Thanks for catching this error. I resolved the problem in th latest push, and added some tests that help it out. The code is kinda ugly still, so I might refactor it as another commit on top of this. I'm still not 100% sure what the right name for this feature is (and whether it makes sense to have an enum or a boolean for it).